src/entity.ts

```diff
@@ -607,6 +591,38 @@ export namespace entity {
       return valueProto;
     }
 
+    if (typeof value === 'number') {
+      const integerOutOfBoundsWarning =
+        "'IntegerOutOfBoundsWarning: the value for '" +
+        property +
+        "' property is outside of bounds of a JavaScript Number.\n" +
+        "Use 'Datastore.int(<integer_value_as_string>)' to preserve accuracy during the upload.";
+
+      const typeCastWarning =
+        "TypeCastWarning: the value for '" +
+        property +
+        "' property is a JavaScript Number.\n" +
+        "Use 'Datastore.int(<integer_value_as_string>)' or " +
+        "'Datastore.double(<double_value_as_string>)' to preserve consistent " +
+        'Datastore types during the upload.';
```

Comment:
I would suggest to change "during the upload." for "in your database".

Also it would be good to have some documentation to refer to.

Reply:
Sure. I will update the other warning to match (this was phrased this way to be consistent)